### PR TITLE
Removing libnice

### PR DESCRIPTION
**Pull request by FelonEkonom**

### CODE REVIEW

File: include/stun.hrl [hunk 1]
```diff
@@ -64,34 +64,19 @@
...
-         'REALM',
-         'REQUESTED-ADDRESS-FAMILY',
-         'REQUESTED-TRANSPORT',
-         'SOFTWARE',
-         'UNKNOWN-ATTRIBUTES' = [],
-         'USERNAME',
-         'XOR-MAPPED-ADDRESS',
-         'XOR-PEER-ADDRESS' = [],
-         'XOR-RELAYED-ADDRESS'}).
+         magic = ?STUN_MAGIC :: non_neg_integer(), trid = 0 :: non_neg_integer(),
```
Comment (mickel8) [inline include/stun.hrl:73]: I don't think it looks better now. Either don't format this file or set formatter in such a way it doesn't mess this up. Please check other places too

File: src/stun_codec.erl [hunk 2]
```diff
@@ -106,6 +106,7 @@ encode(#stun{class = Class,
                end,
            Data = <<0:2, Type:14, (Len + 24):16, Magic:32, TrID:96, Attrs/binary>>,
            MessageIntegrity = crypto:mac(hmac, sha, NewKey, Data),
+           %    MessageIntegrity = <<0:160>>,
```
Comment (mickel8) [inline src/stun_codec.erl:109]: If that's not needed then remove it

File: src/turn_starter.erl [hunk 3]
```diff
@@ -40,7 +41,8 @@ start(Secret, Opts) ->
            Hash = crypto:mac(hmac, sha, Secret, User),
            base64:encode(Hash)
         end,
-    PeerPid = proplists:get_value(peer_pid, Opts),
+    Parent = proplists:get_value(parent, Opts),
```
Comment (mickel8) [inline src/turn_starter.erl:44]: Wasn't `peer` more intuitive name than `parent` is?

**Review by mickel8 — CHANGES_REQUESTED** [review]

File: include/stun.hrl (hunk 1 above)
Comment (FelonEkonom) [reply include/stun.hrl]: I have stopped formatting this file, but there are still some other places with record definitions, that are formatted in this messy way. I have also checked alternate Erlang formatters, but this old one, used currently by us, seems to be the best, despite the fact, that is not formatting records in the prettiest way

**Review by mickel8 — APPROVED** [review]

**Pull request merged**